### PR TITLE
SendGridKit Version 1.0.0

### PR DESCRIPTION
**Pull request by dylanshine**
This package is a port of https://github.com/vapor-community/sendgrid-provider based on feedback addressed in https://github.com/vapor-community/sendgrid-provider/pull/32#issuecomment-546926785.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.2
+import PackageDescription
+
+let package = Package(
+    name: "SendGridKit",
```
Comment (Andrewangeta) [inline Package.swift]: Everything looks good to me. Can we update the package name to be `sendgrid-kit`. This falls in line with `stripe-kit`, `google-cloud-kit` as well as `async-http-client`.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# SendGridKit
+
+![Swift](http://img.shields.io/badge/swift-5.2-brightgreen.svg)
+![Vapor](http://img.shields.io/badge/vapor-4.0-brightgreen.svg)
```
Comment (Andrewangeta) [inline README.md]: We can remove the Vapor specific badge.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+SendGridKit is a Swift package used to communicate with the SendGrid API for Server Side Swift Apps.
+
+## Setup
+Add the dependency to Package.swift:
+
+~~~~swift
+dependencies: [
+	...
+	.package(url: "https://github.com/vapor-community/SendGridKit.git", from: "1.0.0")
```
Comment (Andrewangeta) [inline README.md]: we can update the url to `sendgrid-kit` I've already renamed the repo.

**Comment by dylanshine** [comment]
@Andrewangeta Requested changes have been implemented

**Pull request merged**